### PR TITLE
[WIP] SPD page

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR introduce the SPD page and badge.
Right now, this is only the skeleton.

### CODE REVIEW

File: udata_gouvfr/views.py [hunk 1]
```diff
@@ -9,7 +9,8 @@
 from udata.i18n import I18nBlueprint
 from udata.sitemap import sitemap
 
-from .models import DATACONNEXIONS_5_CANDIDATE, C3, NECMERGITUR, OPENFIELD16
+from .models import DATACONNEXIONS_5_CANDIDATE
+from .models import C3, NECMERGITUR, OPENFIELD16, SPD
```
Comment (davidbgk) [inline udata_gouvfr/views.py]: New way to do multi-line imports?

File: udata_gouvfr/views.py [hunk 2]
```diff
@@ -167,6 +168,12 @@ def openfield16():
                         nb_displayed_datasets=NB_DISPLAYED_DATASETS)
 
 
+@blueprint.route('/spd')
+def spd():
+    datasets = Dataset.objects(badges__kind=SPD)
```
Comment (davidbgk) [inline udata_gouvfr/views.py]: Missing `.visible()`? What about relevant order?

File: udata_gouvfr/tests.py [hunk 3]
```diff
@@ -341,6 +341,21 @@ def test_render_with_data(self):
...
 
+class SpdTest(FrontTestCase):
+    settings = GouvFrSettings
+
+    def test_render_without_data(self):
+        response = self.client.get(url_for('gouvfr.spd'))
+        self.assert200(response)
+
+    def test_render_with_data(self):
+        for i in range(3):
```
Comment (davidbgk) [inline udata_gouvfr/tests.py]: Is one not enough?

File: udata_gouvfr/models.py [hunk 4]
```diff
@@ -29,9 +29,11 @@
 C3 = 'c3'
 NECMERGITUR = 'nec'
 OPENFIELD16 = 'openfield16'
+SPD = 'spd'
 Dataset.__badges__[C3] = _('C³')
 Dataset.__badges__[NECMERGITUR] = _('Nec Mergitur')
 Dataset.__badges__[OPENFIELD16] = 'Openfield 16'
+Dataset.__badges__[SPD] = _('Public Data Service')
```
Comment (davidbgk) [inline udata_gouvfr/models.py]: Is that the way it is translated? If no translation is possible, let's keep it in French.

File: udata_gouvfr/tests.py (hunk 3 above)
Comment (noirbizarre) [inline udata_gouvfr/tests.py]: One no, at least two to test a list iteration.
I'm used to define 3 elements (this is the cas in every listing test), but I can set it to 2.

File: udata_gouvfr/models.py (hunk 4 above)
Comment (noirbizarre) [inline udata_gouvfr/models.py]: I don't know, this is a draft, I'll let @odtvince translate it properly ;)

File: udata_gouvfr/templates/spd.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    <div class="container">
+        <div class="row">
+            <article class="col-xs-12">
+                <h1>Service public de la donnée : des données sur lesquelles vous pouvez compter</h1>
+
+                <p class="text-justify lead">Le service public de la donnée créé par la loi pour une République numérique vise à mettre à disposition, en vue de faciliter leur réutilisation, les jeux de données de référence qui présentent le plus fort impact économique et social. Il s’adresse principalement aux entreprises et aux administrations pour qui la disponibilité d’une donnée de qualité est critique. Les producteurs et les diffuseurs prennent des engagements auprès de ces utilisateurs. La mission Etalab est chargée de la mise en oeuvre et de la gouvernance de ce nouveau service public. Elle référence l’ensemble des données concernées sur cette page.</p>
+
+                <h2>Les données de référence</h2>
+                
+                <p>A ce jour, neuf bases de données, qui couvrent un large champ thématique ont été identifiées comme des données de référence.</p>
```
Comment (davidbgk) [inline udata_gouvfr/templates/spd.html]: `À` :)

File: udata_gouvfr/templates/spd.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+{% block content %}
+{% cache cache_duration, 'spd', g.lang_code %}
+<section class="content">
+    <div class="container">
+        <div class="row">
+            <article class="col-xs-12">
+                <h1>Service public de la donnée : des données sur lesquelles vous pouvez compter</h1>
+
+                <p class="text-justify lead">Le service public de la donnée créé par la loi pour une République numérique vise à mettre à disposition, en vue de faciliter leur réutilisation, les jeux de données de référence qui présentent le plus fort impact économique et social. Il s’adresse principalement aux entreprises et aux administrations pour qui la disponibilité d’une donnée de qualité est critique. Les producteurs et les diffuseurs prennent des engagements auprès de ces utilisateurs. La mission Etalab est chargée de la mise en oeuvre et de la gouvernance de ce nouveau service public. Elle référence l’ensemble des données concernées sur cette page.</p>
```
Comment (davidbgk) [inline udata_gouvfr/templates/spd.html]: Can you reformat on multiple lines to ease the review and future diffs?

File: udata_gouvfr/templates/spd.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+                    {% include theme('dataset/card.html') %}
+                    </li>
+                    {% endfor %}
+                </ul>
+                
+                <h2>L’organisation du service public de la donnée</h2>
+
+                <p>Elle est déterminée par le décret d’application 2017-331 du 14 mars 2017 relatif au service public de mise à disposition des données de référence. 
+
+                <p>Les producteurs produisent la donnée de référence et documentent les métadonnées,<br>ils traitent les remontées des utilisateurs ou le cas échéant, orientent les utilisateurs vers le service compétent pour traiter ces demandes,<br>ils prennent des engagements sur la mise à jour des données,<br>ils désignent les diffuseurs pour chacun des jeux de données de référence.</p>
```
Comment (davidbgk) [inline udata_gouvfr/templates/spd.html]: Are all these `<br>` useful on the web?

File: udata_gouvfr/templates/spd.html [hunk 8]
```diff
@@ -0,0 +1,68 @@
+{% extends theme("base.html") %}
+
+{% set meta = {
+    'title': _('Reference Data'),
+    'description': "Service Public de la Donnée de Référence",
```
Comment (davidbgk) [inline udata_gouvfr/templates/spd.html]: Don't we want to translate the description too?

File: udata_gouvfr/templates/spd.html (hunk 7 above)
Comment (noirbizarre) [inline udata_gouvfr/templates/spd.html]: What about a bullet point list ?

File: udata_gouvfr/tests.py (hunk 3 above)
Comment (davidbgk) [inline udata_gouvfr/tests.py]: Alright, that was just to speed up tests, nevermind.

**Pull request merged**